src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -18,6 +18,7 @@
 import com.google.api.client.util.Lists;
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.Inject;
+import io.netty.util.concurrent.SingleThreadEventExecutor;
```

Comment:
Could you check all the files you've changed/added and make sure that there are no unused imports, please?

I think this is a sign that we need some sort of coding standards in the backend. Is there a ticket for that already?